src/EA.Weee.Web/Areas/AatfReturn/Controllers/SelectReportOptionsController.cs

```diff
@@ -57,6 +58,11 @@ public virtual async Task<ActionResult> Index(Guid organisationId, Guid returnId
                     ReturnData = await client.SendAsync(User.GetAccessToken(), new GetReturn(returnId, false))
                 });
 
+                if (!QuarterHelper.IsOpenForReporting(viewModel.ReturnData.QuarterWindow))
```

Comment:
Possible to add these checks in the ValidateReturnCreatedActionFilter attribute or add new attribute if necessary, same for all controllers, should do the check in the attribute